### DISCUSSION TITLE
Do I need gnustep to invoke the menu or is there some keyboard shortcut?

### DISCUSSION THREAD

**Discussion by wesbluemarine** [roblillack/fsviewer#12] [Q&A]
![screenshot_2024-06-21_at_20:42:26](https://github.com/roblillack/fsviewer/assets/14043560/a893d23d-856c-42d2-a8b1-e7a0e48cea54)

**Comment by roblillack** [comment]
Hey @wesbluemarine, this looks a lot like #4 which is ultimately an issue with the WindowMaker default configuration (see roblillack/wmaker#4). Can you check if your `WMWindowAttributes` contains the offending setting somewhere?
